src/main/java/Duke.java

```diff
@@ -1,10 +1,84 @@
+import java.util.*;
+import tasks.*;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String greeting = " Hello! I'm Duke\n" + " What can I do for you?";
+        String bye = "Bye. Hope to see you again soon!";
+        System.out.println(greeting);
+
+        boolean running = true;
```

Comment:
Boolean variables/methods should be named to sound like booleans
maybe canRun is a better naming